### DISCUSSION TITLE
post/learning-dependency-issues-with-my-library/

### DISCUSSION THREAD

**Discussion by giscus** [jgjgill/blog#31] [Comments]
# post/learning-dependency-issues-with-my-library/

직접 라이브러리를 만들어 사용해보면서 겪은 의존성 이슈에 관한 내용을 다룹니다.

https://jgjgill.com/post/learning-dependency-issues-with-my-library/

**Comment by BaeJi77** [comment]
내용이 너무 재미있네요. 잘 읽었습니다!

**Reply by jgjgill** [reply]
재미있게 읽어주셔서 감사합니다..! 🙇‍♂️
